components/toast/Toast.js

```diff
@@ -54,7 +54,11 @@ class Toast extends PureComponent {
 
   renderCustomLink = () => {
     const { link } = this.props;
-    return link && <TextBody>{React.cloneElement(link, { className: theme['action-link'] })}</TextBody>;
+    return (
+      link && (
+        <TextBody>{React.cloneElement(link, { className: cx(link.props.className, theme['action-link']) })}</TextBody>
+      )
+    );
```

Comment:
Really small and not blocking, but redundant parentheses:
```js
    return link && (
        <TextBody>{React.cloneElement(link, { className: cx(link.props.className, theme['action-link']) })}</TextBody>
      );
```

Reply:
I checked this out and this was added by Prettier when I committed this. I guess it's mainly because the line would have gotten too long.

 :ok: by @Kemosabert on 29/06/2018 09:21:25